### ISSUE TITLE
Recaptcha javascripts are being included twice

### ISSUE THREAD

**Issue by LucaGallinari** [magespecialist/m2-MSP_ReCaptcha#28]
We are using this module on Magento 2.2.3 and as you can see from the snapshot below the scripts are included two times.

<img width="887" alt="screen shot 2018-04-23 at 11 12 26" src="https://user-images.githubusercontent.com/5822554/39419045-174e8da0-4c5e-11e8-9973-278a25c1de32.png">

Do you experience the same problem?

**Comment by LucaGallinari** [comment]
Good point, it's included twice only on pages that really use the recaptcha, on all other pages it's included only once.

**Comment by phoenix128** [comment]
But wait... I did not notice it before.
Our module is not adding any script in the head seaction. It is using requirejs to load it.
Are you sure this issue is not coming from other modules?

**Comment by LucaGallinari** [comment]
I don't know how M2 handle javascripts included by requirejs but i've just seen that those entries on the head block are inserted dynamically. And as you can see from my first comment screenshot it's the require.js script that is loading them.

I tried even with a clean M2.2.2 project and i got the same result:

![screen shot 2018-06-05 at 11 51 30](https://user-images.githubusercontent.com/5822554/40968877-dbf4fad0-68b6-11e8-87e0-9eb8bb089abc.png)

**Comment by phoenix128** [comment]
Never mind, you are right. It is a module issue. Just found it

**Comment by phoenix128** [comment]
Please upgrade to 1.4.11

**Comment by dan-ding** [comment]
I don't think this is fixed

**Comment by phoenix128** [comment]
Hello @dan-ding do you have any issue?

**Comment by dan-ding** [comment]
I was seeing it @phoenix128 , yes.
At first I thought it was because the default captcha was still enabled, but now I'm unsure to why.

I had to remove the extension as it wasn't firing correctly all the time. Sometimes it would, others not. If I get back to trying this out I'll report back with what I find.
